src/Cody.VisualStudio/CodyPackage.cs

```diff
@@ -216,6 +216,12 @@ private async void AuthorizationDetailsChanged(object sender, EventArgs eventArg
 
                 UpdateCurrentWorkspaceFolder();
 
+                if (!status.Authenticated && UserSettingsService.LastTimeAuthorized)
+                    await ShowToolWindowWhenLoggedOut();
+
+                if (status.Authenticated)
```

Comment:
How `UserSettingsService.LastTimeAuthorized` can be set to `false`?
Shouldn't that be:

`UserSettingsService.LastTimeAuthorized = status.Authenticated`

?